test_unit

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
```

Comment:
Where is this file used?

Reply:
I see it mentioned in README... Nice!

Reply:
I wanted to separate C++ and Python tests in the Github actions for the purpose of easier debugging, however included this script for anyone wanting easy check on all the modules in the repository. I have put the same information in the README file.